flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/delegation/StreamPlanner.scala

```diff
@@ -129,6 +129,13 @@ class StreamPlanner(
     sb.append("== Physical Execution Plan ==")
     sb.append(System.lineSeparator)
     sb.append(executionPlan)
+
+    if (extraDetails.contains(ExplainDetail.JSON_EXECUTION_PLAN)) {
+      sb.append(System.lineSeparator)
+      sb.append("== Streaming Execution Plan ==")
+      sb.append(System.lineSeparator)
+      sb.append(streamGraph.getStreamingPlanAsJSON)
+    }
```

Comment:
I have discussed with @godfreyhe . We think that we don't need to add a new entry for the "Streaming Execution Plan". Because it is description the same plan with "Physical Execution Plan", just in different format. 

Thus, we suggest to replace the content of "Physical Execution Plan" using the json string when `ExplainDetail` is in `JSON_EXECUTION_PLAN`.

Reply:
Okay, I will update it and add some test cases.